### PR TITLE
Better return message for bitcoin-s-cli stop

### LINKED ISSUE #1768: Better return message for bitcoin-s-cli stop
This is what gets returned when it works:
```
$ ./bitcoin-s-cli stop
Connection refused (Connection refused)
```

### PR DESCRIPTION
**Pull request by benthecarman**
Closes #1768 

image

### CODE REVIEW

File: app/server/src/main/scala/org/bitcoins/server/NodeRoutes.scala [hunk 1]
```diff
@@ -22,7 +23,14 @@ case class NodeRoutes(node: Node)(implicit system: ActorSystem)
         val nodeStopping = node.stop().map { _ =>
           Server.httpSuccess("Node shutting down")
         }
-        system.terminate()
+
+        val shutdownRunnable = new Runnable {
```
Comment (Christewart) [inline app/server/src/main/scala/org/bitcoins/server/NodeRoutes.scala:27]: Why is this needed? Otherwise we terminate the system before propagating the http success?

**Review by Christewart — APPROVED** [review]

File: app/server/src/main/scala/org/bitcoins/server/NodeRoutes.scala (hunk 1 above)
Comment (benthecarman) [reply app/server/src/main/scala/org/bitcoins/server/NodeRoutes.scala:27]: Yeah, we would queue up the message to send, but then shutdown. This way we give ourselves enough time to send the message
